### DISCUSSION TITLE
Discord for kient / code example sharing?

### DISCUSSION THREAD

**Discussion by b3ck** [zSoulweaver/kient#10] [General]
Looking for a place that may have some code examples, know of any by chance?

**Comment by zSoulweaver** [comment]
I don't believe there is much around, just due to Kick's infancy. On top of that, this library is also very young, missing a lot, and is built on top of a massive workaround to get past Kick's anti-bot measures, it's really not in a spot that I could actively recommend to you and promise you won't have issues. 

Maybe once I can flesh it out and if it picks up in popularity amongst developers I'll look into a discord for support and developer engagement, but at the moment I'm happy just using GitHub to handle any discussions or questions.

**Reply by b3ck** [reply]
All good, was just seeing if there was anything available, but I can for sure keep everything on GitHub, also thanks to your help I figured out how to log messages to the console, I put my working code in my final comment, hopefully it will give people a leg up.
